src/_TextInputWrapper.tsx

```diff
@@ -25,45 +43,50 @@ type StyledWrapperProps = {
   hasIcon?: boolean
   block?: boolean
   contrast?: boolean
+  validationStatus?: 'error' | 'warning'
```

Comment:
Heads up: I defined this as a type in utils/types in one of my PRs: https://github.com/primer/react/pull/1611/files#diff-410797b1c4259d74820358a792a62f83cfc7a1c0a0eb26226c89e4621b88d42cR1

Reply:
I didn't include `'warning'` because I couldn't figure out where we use it, but I'm fine with keeping it.

Reply:
Does that mean I have to handle `success` case in `TextInput`?

Reply:
We don't have a special style for inputs with a validation status of `success`. We just style any success message we might show.

For example:
<img width="177" alt="Screen Shot 2021-12-02 at 8 35 57 AM" src="https://user-images.githubusercontent.com/2313998/144432380-596f0826-ef4f-4a84-af6a-19e02b2715a5.png">

Reply:
Isn't it unnecessary for me to use that type then since it has `success` in it too.